modules/custom-status/lib/components/modals/create-edit-custom-status-modal.js

```diff
@@ -34,7 +46,7 @@ export default function CreateEditCustomStatusModal( { customStatus, onCancel, o
 		const data = {
 			name,
 			description,
-			is_review_required: isReviewRequired,
+			required_user_logins: requiredUserLogins,
```

Comment:
Note that the user restrictions component only cares about user logins, and sends these to the REST endpoint. The REST endpoint then converts these to user IDs for storage. User logins are there for users to understand, and user IDs are better for consistency when stored.